Library/Formula/minizinc.rb

```diff
@@ -1,53 +1,29 @@
-require 'formula'
-
 class Minizinc < Formula
-  homepage 'http://www.minizinc.org'
-  url 'http://www.minizinc.org/downloads/release-1.6/minizinc-1.6-x86_64-apple-darwin.tar.gz'
-  sha1 '71f0e08962eb8bb44c463851f0144c8b006fdb80'
+  homepage "http://www.minizinc.org"
+  head "https://github.com/MiniZinc/libminizinc.git",
+       :branch => "develop"
```

Comment:
I'll make it a single line; I use a large font size for editing so I break lines probably more than is necessary.